### DISCUSSION TITLE
Error while trying to import data from RDS

### DISCUSSION THREAD

**Discussion by juancarlosgzs** [planetscale/discussion#464] [Q&A] [answered]
Hello everyone, I've been trying to do the connection to my RDS production instance and haven't been able to do it so far. After overcoming all the issues related to the variables I need to have turned on or in an specific state, now I'm receiving the following error:

![image](https://github.com/planetscale/discussion/assets/128992105/bb78ce0d-5b94-4d1a-963f-5f361a77477c)

If I put exactly the same information (host, port, database name, username and password) in other tools such as Datagrip or DBeaver I'm able to access the RDS instance without any kind of issues.

However, I've read somewhere in your docs that, if I'm able to connect with the following command, it should be able to connect from there:

`mysql -u <username> -p <password> -h <host> -P <port> -D <databaseName>`

Sadly, when I do so, for some reason that I cannot figure out (And that's why i'm writing this discussion), I receive the following error:

```
Enter password: xxxxxx
ERROR 1049 (42000): Unknown database <password>???
```

Anyway, if instead use the following command, being the only difference not specifying the password itself:

`mysql -u <username> -p -h <host> -P <port> -D <databaseName>`

I'm actually able to connect successfully from the console so idk if that could be the issue when attempting to connect from the import tool.

Any help is, of course, greatly appreciate it, since I'm in a dead end at this point after trying to find the reason by myself.

**Comment by jgreet** [comment, marked as answer]
Could you open a ticket for your import and let us know the organization you're creating the database in? I suspect the importer is connecting to your database OK but failing to parse something in the origin database's schema.

https://support.planetscale.com/hc/en-us/requests/new

-- -

Separately, when using the `mysql` client don't leave a space between `-p` and the password text.

https://dev.mysql.com/doc/refman/8.0/en/mysql-command-options.html#option_mysql_password

>The password value is optional. If not given, [mysql](https://dev.mysql.com/doc/refman/8.0/en/mysql.html) prompts for one. If given, there must be no space between [--password=](https://dev.mysql.com/doc/refman/8.0/en/mysql-command-options.html#option_mysql_password) or -p and the password following it. If no password option is specified, the default is to send no password.

**Reply by juancarlosgzs** [reply]
Thank you very much for such a quick response, sorry for the delay. Today I tried again and suddenly it just worked! Had to remove all the foreign keys first (Thought the migration tool would just ignore them from the schemas) and it's working smooth so far.  In relation to the command itself, you were right, thanks for pointing it out, lesson learned!

**Reply by jgreet** [reply]
Happy to hear it! We rolled out an update yesterday to address a bug you were hitting with the connection test. Sounds like the fixed worked.
